src/org/pcic/DockerUtils.groovy

```diff
@@ -1,37 +1,54 @@
 package org.pcic
 
 import org.pcic.util.Utils
+import org.pcic.GitUtils
+import org.pcic.util.Global
+
 
 class DockerUtils implements Serializable {
     def steps
+    def utils
 
     DockerUtils(steps) {
         this.steps = steps
+        this.utils = new Utils(steps)
     }
 
-    void removeImage(String imageName) {
-        steps.sh(script: "docker rmi ${imageName}")
+    void removeImage(String image) {
+        steps.sh(script: "docker rmi ${image}")
```

Comment:
I kinda like `imageName`; an image and its name are distinct things. That said, if this is more in line with your general pattern of identifiers, then stick with `image`.

Reply:
The reason I changed it to `image` from `imageName` is because it doesn't necessarily have to be the name of the image (it could be the `image_id` as well).